### ISSUE TITLE
Make sure to ignore() component instantiation.

### ISSUE THREAD

**Issue by DesignByOnyx** [bitovi/ylem#114]
I have a VM that was reading and setting some values during init(). The problem was, whenever those values changed later it was causing the parent component to rerender. The reason this was happening was because the parent component was still "listening" when the child component was initialized. The solution was to wrap my init() function in `ObservationRecorder.ignore(...)`. I think RVM should take care of this for us.

**Comment by DesignByOnyx** [comment]
I think it _is_ working the way you describe - it appears only one component is listening at a time. I just think the child components are constructed during the parent render, so any "reads" which happen during construction/init are picked up by the parent.

https://gist.github.com/DesignByOnyx/e45696b579c762756c3bdf1093e8eb92

**Comment by DesignByOnyx** [comment]
I just ran into another scenario where this was necessary - again during component instantiation.

In this case, I have a custom form component which has a "value" property. This property has a setter which reads certain properties and updates others. I was having a weird issue with my component where computed properties which were read during initial instantiation were never updated later. It was a really weird error and I can only describe the issues I was having as "head scratchers". When I wrapped the viewmodel instantiation in an "ignore" (inside RVM), all my problems went away and my component came back to life.
